adapters/Nend/NendAdapter/nativeads/GADMAdapterNendNativeAd.m

```diff
@@ -0,0 +1,134 @@
+//
+//  GADMAdapterNendNativeAd.m
+//  NendAdapter
+//
+//  Copyright © 2019 FAN Communications. All rights reserved.
+//
+
+#import "GADMAdapterNendNativeAd.h"
+#import "GADMAdapterNend.h"
+#import "GADMAdapterNendConstants.h"
+#import "GADNendNativeAdLoader.h"
+
+@interface GADMAdapterNendNativeAd () <NADNativeDelegate>
+
+@property(nonatomic, strong) NADNative *nativeAd;
+@property(nonatomic, strong) GADNativeAdImage *mappedIcon;
+@property(nonatomic, strong) NSArray *mappedImages;
+@property(nonatomic, strong) UILabel *advertisingExplicitlyView;
+
+@end
+
+@implementation GADMAdapterNendNativeAd
+
+- (instancetype)initWithNormal:(NADNative *)ad
+                          logo:(nullable GADNativeAdImage *)logo
+                         image:(nullable GADNativeAdImage *)image {
+    self = [super init];
+    if (self) {
+        _nativeAd = ad;
+        _nativeAd.delegate = self;
+        _advertisingExplicitlyView = [UILabel new];
+        _advertisingExplicitlyView.text = [_nativeAd prTextForAdvertisingExplicitly:NADNativeAdvertisingExplicitlyPR];
+        
+        if (logo) {
+            _mappedIcon = logo;
+        }
+        if (image) {
+            _mappedImages = [NSArray arrayWithObject:image];
+        }
+    }
+    return self;
+}
+
+- (BOOL)hasVideoContent {
+    return false;
+}
+
+- (UIView *)mediaView {
+    return nil;
+}
```

Comment:
This cannot be nil. Adapter must create a UIImageView with contents of the first image if it doesn't have a video.